app/Http/Controllers/PostController.php

```diff
@@ -56,7 +56,8 @@ public function store(CreatePostRequest $request)
      */
     public function show(Post $post)
     {
-        //
+        // for some reason, admin.posts.show doesn't work, but I may just be dumb
```

Comment:
yes because view path is `posts/show` not `admin/posts/show` obv relative to view folder in `resources/views` . Also view has nothing to do with route path. They are similiar because they serve same purpose in this case, but if I really wanted to I could have a route `/lolwtf` call a view `omg.cool.view.dude`